### PR TITLE
bump to v0.0.6

### PR DESCRIPTION
**Pull request by mpmdean**
This is an ultra-naive attempt to update edrixs to v0.0.6
FAO @stuartcampbell @mrakitin

### CODE REVIEW

**Comment by mpmdean** [comment]
> You might want to reset the build number back to 1 as well.

I just changed the build. I was basing what I was doing on the example I found on conda-forge

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml

Is there a better reference for what the build number is doing?

**Comment by mrakitin** [comment]
Actually, the build bumber is recommended to be reset to 0 when a new version is built. That way multiple builds can be performed for the same version if needed (for example, wrong dependencies are listed or some of them are missing).

**Comment by mrakitin** [comment]
@mpmdean, please check this guidelines from the conda-forge community regarding the build numbers: https://conda-forge.org/docs/maintainer/updating_pkgs.html#bumping-the-build-number.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 2
+  number: 1
```
Comment (mrakitin) [inline recipe/meta.yaml:14]:
```suggestion
  number: 0
```

**Comment by mrakitin** [comment]
Also, an official conda-build documentation regarding the build number: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string.

File: recipe/meta.yaml (hunk 1 above)
Comment (mpmdean) [reply recipe/meta.yaml]: I just changed build number to 0

**Comment by mpmdean** [comment]
Thanks

**Pull request merged**